docs/source/notebooks/Delineation_workflow.ipynb

```diff
@@ -13,24 +13,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Cookie-cutter template required to connect to the PAVICS-Hydro Raven WPS server\n",
+    "\n",
     "from birdy import WPSClient\n",
     "import numpy as np\n",
     "import matplotlib.pyplot as plt\n",
     "import geopandas as gpd\n",
     "import json\n",
-    "import os"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "import os\n",
+    "\n",
     "# Set environment variable RAVEN_WPS_URL to \"http://localhost:9099\" to run on the default local server\n",
     "# url = os.environ.get(\"RAVEN_WPS_URL\", \"https://pavics.ouranos.ca/twitcher/ows/proxy/raven/wps\")\n",
     "url = os.environ.get(\"RAVEN_WPS_URL\", \"http://localhost:9099\")\n",
```

Comment:
I know it's not changed in this PR but can you restore the previous good line pointing to Raven on PAVICS?  You can still test against your local Raven if you set the env var `RAVEN_WPS_URL="http://localhost:9099"` when lauching whatever the command to run test.

Reply:
Done!